backend/apps/issues/serializers.py

```
@@ -24,3 +25,42 @@ def create(self, validated_data):
+        model = Bounty
+        fields = [
+            "id",
+            "title",
+            "description",
+            "xp_reward",
+            "status",
+            "claimed_by",
+            "claimed_by_username",
+            "created_at",
+            "updated_at",
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)